guide/message-filtering/passed_student_filter_service.bal

```diff
@@ -1,30 +1,31 @@
 import ballerina/http;
 //import ballerinax/docker;
-//import ballerinax/kubernetes;
-
-// @docker:Config {
-//     registry:"ballerina.guides.io",
-//     name:"passed_student_filter_service",
-//     tag:"v1.0"
-// }
-
-//  @docker:Expose{}
+//
+//@docker:Config {
+//    registry: "ballerina.guides.io",
+//    name: "passed_student_filter_service",
+//    tag: "v1.0"
```

Comment:
Is the ballerina runtime image enough for this BBG? Otherwise we have to specifically add the baseImage config